sdk/go/common/constant/exec_kind.go

```diff
@@ -11,3 +11,7 @@ const ExecKindAutoInline = "auto.inline"
 // ExecKindCLI is a flag used to indentify a command as originating
 // from the CLI using a traditional Pulumi project.
 const ExecKindCLI = "cli"
+
+// ExitStatusLoggedError is the exit status to indicate that a pulumi program
+// has failed, but successfully logged an error message to the engine
+const ExitStatusLoggedError = 213
```

Comment:
I just recalled that when working on the other languages almost a month ago, I reserved 32 as an exit status. We should reuse this number for consistency's sake.

https://github.com/pulumi/pulumi/blob/490be1cc52673b7afe26cc4365e199005ed65d4c/sdk/python/cmd/pulumi-language-python/main.go#L67-L70